9_report/src/fig_input_data.R

```diff
@@ -0,0 +1,48 @@
+fig_input_data <- function(fig_ind, preds_ind, remake_file, config_file){
+  #discharge, concentration, loads
+
+  site = '05465500'
+  model = 'long1'
+  ref_date = '2017-05-09'
+
+  preds <- readRDS(sc_retrieve(preds_ind, remake_file)) %>%
+    dplyr::filter(ref_date == '2017-05-09', # dplyr::filter doesn't recognize objects??
+                  model_range == 'long1',
+                  Site == '05465500')
+
+  # example of input data for a forecast
+  eList <- prep_inputs(nwis_site = site,
+              nwm_model = model,
+              ref_date = ref_date,
+              site_info_ind = '1_data/out/site_info.tsv.ind',
+              nwis_data_ind = '2_munge/out/agg_nwis.rds.ind',
+              nwm_retro_ind = '2_munge/out/agg_nwm_retro.rds.ind',
+              nwm_forecast_ind = '2_munge/out/agg_nwm_long1.rds.ind',
+              remake_file = '3_forecast.yml')
+
+  xlim = range(c(eList$Daily$Date, eList$Sample$Date, preds$Date))
+
+  # flow
+  g1 <- ggplotGrob(ggplot(data = eList$Daily, aes(x = Date, y = Q)) +
+    geom_line() + theme_classic() + xlim(xlim) +
+      theme(axis.title.x = element_blank()))
+
+  #concentration
+  g2 <- ggplotGrob(ggplot(data = eList$Sample, aes(x = Date, y = ConcAve)) +
```

Comment:
we could think about using lollipops or randomized residuals for censored data, a la https://owi.usgs.gov/blog/rResid/. @ldecicco-USGS, want to weigh in on this?

Reply:
I don't have a favorite, but it would be a good touch to flag censored data. Could do a simple color a la https://owi.usgs.gov/blog/plotFlowConc/
You could also just use the `EGRET` plot functions. `multiPlotDataOverview` is a nice intro plot to the data.